### ISSUE TITLE
Self-Hosted shows message about running in develop mode

### ISSUE THREAD

**Issue by Vrtak-CZ** [SolidStateGroup/bullet-train-api#323]
I just wonder if is there any reason for showing "You are using the develop environment." on self-hosted flagsmith? Looks like it's feature served by flagsmith it self.

```json
{
        "id": 35671,
        "feature": {
            "id": 7168,
            "name": "butter_bar",
            "created_date": "2021-02-10T20:03:43.348556Z",
            "description": "Show html in a butter bar for certain users",
            "initial_value": null,
            "default_enabled": false,
            "type": "STANDARD"
        },
        "feature_state_value": "<strong>\nYou are using the develop environment.\n</strong>",
        "enabled": true,
        "environment": 23,
        "identity": null,
        "feature_segment": null
    }
```

I understand that this issue https://github.com/Flagsmith/flagsmith/issues/122 will probably solve this. But I'm not sure if there is another reason for this message?

**Comment by dabeeeenster** [comment]
Hi @Vrtak-CZ - yes this is being worked on and will be fixed in #122 - its just the defaults when you hit our SaaS Flagsmith flags which is being resolved by #122

**Comment by dabeeeenster** [comment]
Fixed in docker-compose
